rllib/evaluation/sampler.py

```diff
@@ -306,6 +306,14 @@ def get_batch_builder():
     def new_episode():
         episode = MultiAgentEpisode(policies, policy_mapping_fn,
                                     get_batch_builder, extra_batch_callback)
+        # Call each policy's Exploration.on_episode_start method.
+        for p in policies.values():
+            p.exploration.on_episode_start(
+                policy=p,
+                environment=base_env,
+                episode=episode,
+                tf_sess=getattr(p, "_sess", None))
```

Comment:
```suggestion
                tf_sess=tf_sess)
```

(It's already available as a local variable here passed to the sampler.)